### PR TITLE
feat(ExchangeDetailViewController): IOS-1393 add refund button to expired trades.

### PR DESCRIPTION
**Pull request by woudini**
## Objective

Add refund button to expired trades and link to support ticket request page.

## Description

- Added description sections for expired and failed trades.
- Added button for expired trades to open an `SFSafariViewController` to the support page.

## How to Test

- Go to Exchange
- See that all trade detail views are viewable from the `ExchangeListViewController`
- See that descriptions are shown for expired and failed trades.
- See that a request refund option is available for expired trades and that it opens the support page.

## Screenshot/Design assessment

simulator screen shot - iphone 5s - 2018-10-04 at 20 46 09
simulator screen shot - iphone 5s - 2018-10-04 at 20 46 03

## Merge Checklist

- [x] The PR uses a title supported by [.changelogrc](https://github.com/blockchain/My-Wallet-V3-iOS/blob/dev/.changelogrc#L6...L69).
- [ ] Areas of technical debt are marked with a `// TICKET:` comment that includes a ticket number.
- [x] All unit tests pass.
- [ ] You have added unit tests.
- [ ] You have added sufficient documentation (descriptive comments).
- [ ] New files added are within the correct directory. (e.g. if a file is required for unit tests to compile, be sure it is added to the tests target.)

### CODE REVIEW

File: Blockchain/Homebrew/Exchange/Controllers/ExchangeDetailViewController.swift [hunk 1]
```diff
@@ -129,14 +130,20 @@ class ExchangeDetailViewController: UIViewController {
...
                 forSupplementaryViewOfKind: UICollectionElementKindSectionHeader,
                 withReuseIdentifier: ExchangeDetailHeaderView.identifier
             )
+
+            let footerNib = UINib(nibName: ActionableFooterView.identifier, bundle: nil)
+            collectionView.register(
+                footerNib,
+                forSupplementaryViewOfKind: UICollectionElementKindSectionFooter,
+                withReuseIdentifier: ActionableFooterView.identifier
+            )
```
[the remark this answers is not in the document]
Comment (woudini) [reply Blockchain/Homebrew/Exchange/Controllers/ExchangeDetailViewController.swift]: Tried this just now but got an exception. `.locked` is using `ExchangeLockedHeaderView` while `.overview` is using `ExchangeDetailHeaderView`

[the remark this answers is not in the document]
Comment (thisisalexmcgregor) [reply Blockchain/Homebrew/Exchange/Controllers/ExchangeDetailViewController.swift]: Oh my bad, I read it too fast.

[the remark this answers is not in the document]
Comment (thisisalexmcgregor) [reply Blockchain/Homebrew/Exchange/Controllers/ExchangeDetailViewController.swift]: Just do the modal presentation then it's good to go.

**Comment by thisisalexmcgregor** [comment]
This is G2G

**Pull request merged**